client/my-sites/stats/stats-subscribers-highlight-section/index.tsx

```diff
@@ -2,47 +2,35 @@ import config from '@automattic/calypso-config';
 import { CountComparisonCard } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
 import QueryMembershipProducts from 'calypso/components/data/query-memberships';
+import useSubscribersOverview from 'calypso/my-sites/stats/hooks/use-subscribers-overview';
 import { useSelector } from 'calypso/state';
 import './style.scss';
 import useSubscribersTotalsQueries from '../hooks/use-subscribers-totals-query';
 
-function useSubscriberHighlights(
-	siteId: number | null,
-	hasAddedPaidSubscriptionProduct: boolean
-) {
+function useSubscriberHighlights( siteId: number | null ) {
 	const translate = useTranslate();
 
-	const { data: subscribersTotals, isLoading, isError } = useSubscribersTotalsQueries( siteId );
+	const {
+		data: subscribersTotals,
+		isLoading: isSubscribersTotalLoading,
+		isError: isSubscribersTotalError,
+	} = useSubscribersTotalsQueries( siteId );
+	const {
+		overviewData,
+		isLoading: isOverviewDataLoading,
+		isError: isOverviewDataError,
+	} = useSubscribersOverview( siteId );
+
+	const isLoading = isSubscribersTotalLoading || isOverviewDataLoading;
+	const isError = isSubscribersTotalError || isOverviewDataError;
 
 	const highlights = [
 		{
 			heading: translate( 'Total subscribers' ),
 			count: subscribersTotals?.total,
-			show: true, // Always show total subscribers.
 			note: 'WordPress.com and Email subscribers excluding subscribers from social media',
 		},
-		{
-			heading: translate( 'Paid subscribers' ),
```

Comment:
I imagine Paid & Free might be pretty important to users with Paid Newsletters, which we might want to keep, don't we @simison?

Reply:
I agree with it, but the layout would be a little tricky. What do you think? @ederrengifo